client/ReviewModule.jsx

```diff
@@ -75,6 +76,64 @@ export default class ReviewModule extends React.Component {
     this.setState({ redVote: redArray, greyVote: greyArray });
   }
 
+  handleDropDown(e) {
+    const { reviews } = this.state;
+    const sortType = e.target.value;
+    let unsorted = reviews.slice();
+    const lowFirst = function compareLowest(a, b) {
+      return a.stars - b.stars;
+    };
+    const highFirst = function compareHighest(a, b) {
+      return b.stars - a.stars;
+    };
+    const earlyFirst = function compareEarliest(a, b) {
+      const yrA = parseInt(a.posted.slice(0, 4), 10);
```

Comment:
There is actually a easier way to do this using javascript build in date time. 
var datea = date.parse(a.posted);
var dateb = date.parse(b.posted);
compare datea and dateb etc. 
you  can read more into the docs if you are interested.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/prototype

Reply:
Wow that is so much better.  Thank you!